ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c

```diff
@@ -66,8 +70,13 @@
 void init_is (void) __attribute__ ((constructor));
 void fini_is (void) __attribute__ ((destructor));
 
-#define RGBA_CHANNEL                   4
-#define TFLITE_DEEPLAB_TOTAL_LABELS    21
+#define TOTAL_LABELS  (21)  /* PASCAL VOC */
+#define RGBA_CHANNEL  (4)
+
+/** @todo Support neon64 to accelerate color mapping */
+#if defined(__aarch64__)
+#define NEON64_ENABLED
```

Comment:
Das ist gut!

(but I'm not sure if NEON is mandatory for all aarch64 machines)

Reply:
In aarch64, NEON is always activated (so, there's no configuration to change this) unlike arm32.